docs/2_1_patching.md

```diff
@@ -11,7 +11,7 @@ The following pages will guide you through using ReVanced Manager to patch apps.
 5. Select the patches you want to apply
 6. Tap the Patch button
 7. Tap on the **Install** button
-   > **Note**: If you are rooted, you can mount the patched app on top of the original app.[^3]
+   > [!Note]: If you are rooted, you can mount the patched app on top of the original app.[^3]
```

Comment:
Above, you do not add : and instead use a newline. This should be done here too

Reply:
```suggestion
   > [!Note]
   > If you are rooted, you can mount the patched app on top of the original app.[^3]
```


Co-authored-by: oSumAtrIX <johan.melkonyan1@web.de>